File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/typescript/ADV-15.ts
console.log("ADV-START");

const xs: number[] = [1, 2, 3, 4, 5];
let iters: number = 0;
for (const x of xs) {
    iters += 1;
    if (x === 2) {
        xs.push(99);
    }
}

console.log("OBS=ITERS:" + String(iters) + "|LEN:" + String(xs.length));
console.log("ADV-END");

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/go/ADV-15.go
package main

import "fmt"

func main() {
	fmt.Println("ADV-START")

	xs := []int64{1, 2, 3, 4, 5}
	iters := 0
	for _, x := range xs {
		iters++
		if x == 2 {
			xs = append(xs, 99)
		}
	}

	fmt.Printf("OBS=ITERS:%d|LEN:%d\n", iters, len(xs))
	fmt.Println("ADV-END")
}
